azure-devops/run_build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       displayName: 'Run vcpkg'
       timeoutInMinutes: 10
       inputs:
-        vcpkgArguments: '@$(vcpkgResponseFile)'
+        vcpkgArguments: 'boost-math:${{ parameters.targetPlatform }}-windows'

```

Comment:
does building only what you need in this job improve perfs? This should happen only when you update vcpkg, then all other jobs should get it built already (hence a no-op) from the `cache` task.

Reply:
As I said, it's a slight improvement but it just feels wasteful to not do it. The difference can be especially noticeable on some slower machines that I tested this on.

Reply:
right, it is needed indeed